script/start-odc.sh

```diff
@@ -54,17 +54,25 @@ function usage() {
     echo "- SERVER_PORT: deprecated environment variable, same as ODC_SERVER_PORT, for compatibility"
 }
 
+function log_info() {
+    echo 1>&2 "$(date +"%Y-%m-%dT%H:%M:%S.%Z") [INFO]" "$*"
```

Comment:
Why do INFO level logs need to be redirected to standard error output as well?

Reply:
good question,  the code copy from where, here we may not redirect stdout to stderr.
fixed now.